helm/hpcc/values.yaml

```diff
@@ -370,7 +370,7 @@ eclagent:
   ## prefix may be used to set a filename prefix applied to any relative filenames used by jobs submitted to this queue
   prefix: hthor
   ## Set to false if you want to launch each workunit in its own container, true to run as child processes in eclagent pod
-  useChildProcesses: false
+  useChildProcesses: true
```

Comment:
Was this intentional?

Reply:
no - changed to get logs I could look at easily.